tools/importer/rules/videos.js

```diff
@@ -0,0 +1,102 @@
+const textElClasses = [
+  'video-title',
+  'video-description',
+  'video-author',
+  'video-website'
+];
+
+export default function createVideosEmbed(main, document) {
+
+  // Youtube - AdobeTV
+  main.querySelectorAll('.video').forEach((video) => {
```

Comment:
It looks as if this is an older component which shouldn't have be used on article pages, instead a dexter based  video component should be supported.

URL - https://helpx.adobe.com/manage-account/using/cancel-subscription.html

<img width="1260" alt="Screenshot 2023-01-12 at 8 57 49 PM" src="https://user-images.githubusercontent.com/4350668/212145881-66ca6375-1239-4b9c-82ea-244edd061349.png">

Reply:
@nkthakur48, I added this rule based on what I've found on this page: https://helpx-internal.corp.adobe.com/content/help/en/adobe-connect/using/meeting-basics.html
There, the "video" component is containing the video embed AND a list of metadata that are displayed under the video.
Should I support this use case (video + metadata fields) or I can only import the video and NOT the metadata fields?